cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/queue/JobQueue.java

```diff
@@ -41,14 +42,7 @@
    * @return the stored Job, or null if there is no Job for the given key.
    */
   @Nullable
-  Job getJob(JobKey jobKey);
-
-  /**
-   * Creates a new Job in the queue or updates an existing Job.
-   *
-   * @param job the new job
-   */
-  void put(Job job);
```

Comment:
Is there a reason this method is removed from the interface? I see it is still present in the JobQueueTable

Reply:
Generally, no service should be writing the Jobs directly to the table. JobQueue has other methods that create a job when required.